python_scripts/03_categorical_pipeline.py

```diff
@@ -276,30 +278,36 @@
 data["native-country"].value_counts()
 
 # %% [markdown]
-# We see that the `Holand-Netherlands` category is occurring rarely. This will
+# We see that the `"Holand-Netherlands"` category is occurring rarely. This will
 # be a problem during cross-validation: if the sample ends up in the test set
 # during splitting then the classifier would not have seen the category during
 # training and will not be able to encode it.
 #
-# In scikit-learn, there are two solutions to bypass this issue:
+# In scikit-learn, there are some possible solutions to bypass this issue:
 #
 # * list all the possible categories and provide it to the encoder via the
 #   keyword argument `categories`;
```

Comment:
```suggestion
#   keyword argument `categories` instead of letting the estimator try to
#   automatically find this list only from the training side of the split when
#   calling `fit`;
```